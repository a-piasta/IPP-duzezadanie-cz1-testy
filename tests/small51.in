addRoad;opksprtajbwpit;vaygwufypbzohnp;24;-14
addRoad;owyhbwccxyosnbiuvdatolpeahalq;wgfungsriqrykfbsbxxc;24;2
extendRoute;50;wgfungsriqrykfbsbxxc
extendRoute;50;penhxihgmexuzclkamo
getRouteDescription;248
removeRoad;wgfungsriqrykfbsbxxc;owyhbwccxyosnbiuvdatolpeahalq
removeRoad;wgfungsriqrykfbsbxxc;owyhbwccxyosnbiuvdatolpeahalq
addRoad;opksprtajbwpit;viwrkyfontcvsbbwl;15;10
removeRoad;vaygwufypbzohnp;opksprtajbwpit
repairRoad;viwrkyfontcvsbbwl;wgfungsriqrykfbsbxxc;-10
removeRoad;wgfungsriqrykfbsbxxc;owyhbwccxyosnbiuvdatolpeahalq
getRouteDescription;203
repairRoad;viwrkyfontcvsbbwl;owyhbwccxyosnbiuvdatolpeahalq;15
newRoute;818;viwrkyfontcvsbbwl;lboojmoqojnfpamnluvxa
newRoute;955;vaygwufypbzohnp;owyhbwccxyosnbiuvdatolpeahalq
extendRoute;955;tbxglzsbqefesosjgguto
newRoute;542;tbxglzsbqefesosjgguto;wgfungsriqrykfbsbxxc
newRoute;814;opksprtajbwpit;tbxglzsbqefesosjgguto
removeRoad;viwrkyfontcvsbbwl;opksprtajbwpit
repairRoad;viwrkyfontcvsbbwl;penhxihgmexuzclkamo;9
removeRoad;wgfungsriqrykfbsbxxc;owyhbwccxyosnbiuvdatolpeahalq
newRoute;335;lboojmoqojnfpamnluvxa;opksprtajbwpit
newRoute;84;penhxihgmexuzclkamo;vaygwufypbzohnp
repairRoad;tbxglzsbqefesosjgguto;opksprtajbwpit;-7
repairRoad;lboojmoqojnfpamnluvxa;penhxihgmexuzclkamo;4
newRoute;492;olwzbpznwbhyafau;wgfungsriqrykfbsbxxc
newRoute;341;nlofuvewlluompwljg;owyhbwccxyosnbiuvdatolpeahalq
addRoad;opksprtajbwpit;tbxglzsbqefesosjgguto;28;-8
extendRoute;335;wgfungsriqrykfbsbxxc
getRouteDescription;492
newRoute;991;penhxihgmexuzclkamo;owyhbwccxyosnbiuvdatolpeahalq
removeRoad;opksprtajbwpit;viwrkyfontcvsbbwl
getRouteDescription;492
removeRoad;vaygwufypbzohnp;opksprtajbwpit
addRoad;nlofuvewlluompwljg;owyhbwccxyosnbiuvdatolpeahalq;29;7
removeRoad;nlofuvewlluompwljg;owyhbwccxyosnbiuvdatolpeahalq
extendRoute;955;owyhbwccxyosnbiuvdatolpeahalq
repairRoad;owyhbwccxyosnbiuvdatolpeahalq;olwzbpznwbhyafau;-2
addRoad;penhxihgmexuzclkamo;nlofuvewlluompwljg;13;-13
getRouteDescription;814
extendRoute;818;vaygwufypbzohnp
removeRoad;nlofuvewlluompwljg;penhxihgmexuzclkamo
removeRoad;viwrkyfontcvsbbwl;opksprtajbwpit
repairRoad;tbxglzsbqefesosjgguto;lboojmoqojnfpamnluvxa;-9
addRoad;opksprtajbwpit;vaygwufypbzohnp;4;-7
extendRoute;955;tbxglzsbqefesosjgguto
addRoad;wgfungsriqrykfbsbxxc;viwrkyfontcvsbbwl;8;7
extendRoute;341;vaygwufypbzohnp
getRouteDescription;991
extendRoute;542;vaygwufypbzohnp
newRoute;611;viwrkyfontcvsbbwl;olwzbpznwbhyafau
repairRoad;owyhbwccxyosnbiuvdatolpeahalq;penhxihgmexuzclkamo;1
getRouteDescription;698
newRoute;509;tbxglzsbqefesosjgguto;nlofuvewlluompwljg
removeRoad;viwrkyfontcvsbbwl;wgfungsriqrykfbsbxxc
removeRoad;opksprtajbwpit;vaygwufypbzohnp
newRoute;878;owyhbwccxyosnbiuvdatolpeahalq;tbxglzsbqefesosjgguto
repairRoad;viwrkyfontcvsbbwl;tbxglzsbqefesosjgguto;6
extendRoute;955;wgfungsriqrykfbsbxxc
