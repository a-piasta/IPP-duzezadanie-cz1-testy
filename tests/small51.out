1: TAK
2: TAK
3: NIE
4: NIE
5: 
6: TAK
7: NIE
8: TAK
9: TAK
10: NIE
11: NIE
12: 
13: NIE
14: NIE
15: NIE
16: NIE
17: NIE
18: NIE
19: TAK
20: NIE
21: NIE
22: NIE
23: NIE
24: NIE
25: NIE
26: NIE
27: NIE
28: TAK
29: NIE
30: 
31: NIE
32: NIE
33: 
34: NIE
35: TAK
36: TAK
37: NIE
38: NIE
39: TAK
40: 
41: NIE
42: TAK
43: NIE
44: NIE
45: TAK
46: NIE
47: TAK
48: NIE
49: 
50: NIE
51: NIE
52: NIE
53: 
54: NIE
55: TAK
56: TAK
57: NIE
58: NIE
59: NIE
